charmander/stores/example_store.py

```diff
@@ -0,0 +1,30 @@
+"""
```

Comment:
The store manages the persistence of the model to the db. Our stores manage special (e.g. technology-specific) logic for persisting data. The idea is you can swap out the store layer and any persistence specific logic will easily be in one place. We've done this before with `postgres` and `elasticsearch`.